spec/controllers/problems_controller_spec.rb

```diff
@@ -173,6 +173,17 @@
     end
   end
 
+  describe "GET /apps/:app_id/problems/:id/xhr_sparkline" do
+    before do
+      sign_in user
+    end
+
+    it "renders without error" do
+      get :xhr_sparkline, app_id: app.id, id: err.problem.id
+      expect(response).to be_success
```

Comment:
I think we're going to need something a little more robust in terms of testing here.

Reply:
added a bunch of tests for the underlying data in problems_spec.